kubernetes/e2e_test/test_apps.py

```diff
@@ -17,35 +17,37 @@
 import yaml
 
 from kubernetes.client import api_client
-from kubernetes.client.apis import extensions_v1beta1_api
-from kubernetes.client.models import v1_delete_options
+from kubernetes.client.apis import apps_v1_api
 from kubernetes.e2e_test import base
 
 
-class TestClientExtensions(unittest.TestCase):
+class TestClientApps(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
         cls.config = base.get_e2e_configuration()
 
     def test_create_deployment(self):
         client = api_client.ApiClient(configuration=self.config)
-        api = extensions_v1beta1_api.ExtensionsV1beta1Api(client)
+        api = apps_v1_api.AppsV1Api(client)
         name = 'nginx-deployment-' + str(uuid.uuid4())
-        deployment = '''apiVersion: extensions/v1beta1
+        deployment = '''apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: %s
 spec:
   replicas: 3
+  selector:
+	matchLabels:
```

Comment:
nit: fix indentation

Reply:
Fixed. The original indentation looked that way.